modules/ui/org.eclipse.fx.ui.preferences/src/main/java/org/eclipse/fx/ui/preferences/PreferenceUI.java

```diff
@@ -276,11 +276,16 @@ public void setPreferencePageProviders(@Service List<PreferencePageProvider> pro
 		this.providers.setAll(providers);
 	}
 
-	private Predicate<PreferencePageProvider> getFilter() {
+	protected Predicate<PreferencePageProvider> getFilter() {
 		String filter = currentFilter == null ? "*" : "*" + currentFilter + "*"; //$NON-NLS-1$ //$NON-NLS-2$ //$NON-NLS-3$
```

Comment:
I wonder if we not have to provide acceess to the currentFilter value so that subclasses can access it?

Reply:
You're right, I made the field protected.